AUTHORS.md

```diff
@@ -192,9 +192,13 @@
 - Will Monroe
 - Elijah Rippeth
 - Emil Manukyan
+<<<<<<< HEAD
```

Comment:
fixed, sorry about that. And I tried to rebase but I think my branch is way too enmeshed in master at this point.